Cargo.lock

```diff
@@ -415,6 +415,7 @@ dependencies = [
 name = "cargo-platform"
 version = "0.3.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "122ec45a44b270afd1402f351b782c676b173e3c3fb28d86ff7ebfb4d86a4ee4"
```

Comment:
Fixed in https://github.com/facebookincubator/reindeer/commit/eabd1ae3c112bd2c50263802c5c34b4406b84947. This was related to using a patched `cargo` and `cargo-platform` from git to pull in https://github.com/rust-lang/cargo/pull/15653 and https://github.com/rust-lang/cargo/pull/15753.